src/riak_kv_compile_tab.erl

```diff
@@ -71,35 +100,75 @@ is_compiling(BucketType) ->
         compiling_state() | notfound.
 get_state(BucketType) when is_binary(BucketType) ->
     case dets:lookup(?TABLE, BucketType) of
-        [{_,_,_,State}] ->
+        [{_,_,_,_,State}] ->
```

Comment:
I understand, I was just contemplating the consequences if that cleanup doesn’t occur.

-John

> On Apr 4, 2016, at 4:43 PM, Brett Hazen <notifications@github.com> wrote:
> 
> In src/riak_kv_compile_tab.erl <https://github.com/basho/riak_kv/pull/1377#discussion_r58445308>:
> 
> > @@ -71,35 +100,75 @@ is_compiling(BucketType) ->
> >          compiling_state() | notfound.
> >  get_state(BucketType) when is_binary(BucketType) ->
> >      case dets:lookup(?TABLE, BucketType) of
> > -        [{_,_,_,State}] ->
> > +        [{_,_,_,_,State}] ->
> The whole idea is to clean up these before we get into the code. This is the "upgrade once" idea which then in the rest of the code we assume to be already upgraded.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/basho/riak_kv/pull/1377/files/35f4169741052f5c073112ce8b553dc712cb6102#r58445308>